file/file.go

```diff
@@ -397,3 +397,90 @@ func HasPermission(fileName string, permission int) (bool, error) {
 
 	return true, nil
 }
+
+func CountLine(filename string) (uint64, error) {
+	file, err := os.Open(filename)
+	if err != nil {
+		return 0, err
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	var lineCount uint64
+	for scanner.Scan() {
+		lineCount++
+	}
+
+	if err := scanner.Err(); err != nil {
+		return 0, err
+	}
+	return lineCount, nil
+}
+
+func CountLineWithSeparator(separator, filename string) (uint64, error) {
```

Comment:
- `separator` can be directly a `[]byte`
- The function name should be `CountLinesWithSeparator`
- Following the same previous pattern, the same can be applied here making it variadic and returning FileInfo for each processed file